pkg/deploy/envParams.go

```diff
@@ -121,3 +127,34 @@ func ApplyParams(componentPath string, imageParamsMap map[string]string, extraPa
 
 	return nil
 }
+
+func GetReleaseVersion(in *status.ComponentsStatus, defaultManifestPath string, componentName string) (status.ComponentStatus, error) {
+	var componentVersion semver.Version
+	var repositoryURL string
+	var displayName string
+
+	env, err := parseParams(filepath.Join(defaultManifestPath, componentName, ".env"))
```

Comment:
probably should be moved to some common place and exported. Putting GetReleaseVersion() probably not the best.